OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyLogic.cs

```diff
@@ -26,7 +26,8 @@ public class LobbyLogic : ChromeLogic
 	{
 		static readonly Action DoNothing = () => { };
 
-		public MapPreview Map = MapCache.UnknownMap;
+		public MapPreview MapPreview = MapCache.UnknownMap;
+		public Map Map = null;
```

Comment:
Are you sure about this? It shouldn't be any different to what we were already doing - i've just moved the reference from inside the MapPreview to here.